spot_rl_experiments/experiments/skill_test/test_semantic_gaze.py

```diff
@@ -2,19 +2,90 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
-
-import numpy as np
 from spot_rl.envs.skill_manager import SpotSkillManager
 
 if __name__ == "__main__":
     from perception_and_utils.utils.generic_utils import map_user_input_to_boolean
 
-    spotskillmanager = SpotSkillManager(use_mobile_pick=True)
+    spotskillmanager = SpotSkillManager(use_mobile_pick=False, use_semantic_place=True)
     contnue = True
+    object_name = "bottle"  # "penguin plush toy"
     while contnue:
-        spotskillmanager.semanticpick("penguin", "topdown")
-        spotskillmanager.spot.open_gripper()
+        spotskillmanager.spot.stand()
+        spotskillmanager.pick(
```

Comment:
the previous code uses the name `semanticpick` to reflect the fact that this is a newly-trained skill. Here, we are repurposing the old pick skill to have pose correction and pose estimation. Do you think we move the function to semantic pick or pick?